react/src/components/Chat/EndpointSelect.tsx

```diff
@@ -175,6 +179,33 @@ const EndpointSelect: React.FC<EndpointSelectProps> = ({
       onSearch={(v) => {
         setSearchStr(v);
       }}
+      labelRender={({ label, value }) => {
+        return (
+          <>
+            {label}
+            {value && (
+              <Button
```

Comment:
Quick question. Why we prefer to use inline component even less dedicated variables and indepently component. uses. not use wrapped component <EndpointInfo endpoint={selectedEndpoint?.endpoint_id} />

Reply:
Or we can use BAILink instead according to @yomybaby 's suggestion. What do you think?